airgun/entities/task.py

```diff
@@ -20,6 +20,22 @@ def read(self, entity_name, widget_names=None):
         view = self.navigate_to(self, 'Details', entity_name=entity_name)
         return view.read(widget_names=widget_names)
 
+    def set_chart_filter(self, char_name, index=None):
+        view = self.navigate_to(self, 'All')
+        if char_name not in view.widget_names:
+            raise ValueError('Provide correct chart name')
```

Comment:
getattr for attribute that does not exist will raise exception like:
```
AttributeError: type object 'TasksView' has no attribute 'nonexistingchart'
```
Don't you think it's readable enough? Why do you shadow it with your own exception?